docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -121,7 +121,7 @@ A synchronously replicated database ensures that data written in one site is alw
 It also ensures that the next request will not return stale data, independent on which site it is served.
 
 Why synchronous {jdgserver_name} replication?::
-A synchronously replicated {jdgserver_name} ensures that sessions created, updated and deleted in one site are always available on the other site after a site failure and no data is lost.
+A synchronously replicated {jdgserver_name} ensures that cached data cin one site are always available on the other site after a site failure and no data is lost.
```

Comment:
Type, `/cin/in`

Reply:
I'm just checking if you guys are paying attention :rofl: